src/excel-markdown-tables.ts

```diff
@@ -0,0 +1,106 @@
+const ALIGNED_LEFT = "l";
+const ALIGNED_RIGHT = "r";
+const ALIGNED_CENTER = "c";
+
+
+/**
+ * Test fuck
```

Comment:
Test what?

Reply:
Very sorry about that. I thought I'd cleaned that up. I can purge it from the git history if you'd like